admin/metabox/class-metabox.php

```diff
@@ -231,7 +231,13 @@ public function localize_post_scraper_script() {
 			new WPSEO_Post_Metabox_Formatter( $post, array(), $permalink )
 		);
 
-		return $post_formatter->get_values();
+		$values     = $post_formatter->get_values();
+		$post_types = apply_filters( 'wpseo_cornerstone_post_types', WPSEO_Post_Type::get_accessible_post_types() );
```

Comment:
question: should this filter (which is added also in `WPSEO_Cornerstone_Filter->get_post_types()`) be documented?